lib/index.js

```diff
@@ -10,6 +10,9 @@ let instance
 
 export default {
   activate() {
+    greeter = new Greeter()
```

Comment:
This is only used down on L41, why not just define it as a `const` in that `if`?